solidity/package.json

```
@@ -21,8 +21,8 @@
     "lint:fix": "npm run lint:fix:js && npm run lint:fix:sol",
     "lint:js": "eslint ${npm_package_config_eslintPaths}",
     "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}",
```

Would removing `eslintPaths` configuration work here?
```suggestion
    "lint:js": "eslint .",
    "lint:fix:js": "eslint --fix .",
```
See https://github.com/keep-network/keep-core/pull/2244#discussion_r556652567

---

Done a3ab8b7